share/poly.go

```diff
@@ -40,7 +41,11 @@ func (p *PriShare) Hash(s kyber.HashFactory) []byte {
 }
 
 func (p *PriShare) String() string {
-	return fmt.Sprintf("{%d:%p}", p.I, p.V)
+	data, err := p.V.MarshalBinary()
+	if err != nil {
+		panic("couldn't marshal point")
+	}
+	return fmt.Sprintf("{%d:%s}", p.I, hex.EncodeToString(data))
```

Comment:
```suggestion
		return "wrong V-value"
	}
	return fmt.Sprintf("{%d:%x}", p.I, data)
```

Reply:
Both are also fine to me.

Reply:
In fact there is already a `String()` function for a `kyber.Scalar`.

So why not doing: ? 

```
return fmt.Sprintf("{%d:%s}", p.I, p.V)
```

Reply:
I have already talked about this with @ineiti before, as seen in #455. 

> As p.V is a kyber.Scalar, I'm not sure that the String method is always meaningful. Probably the best would be to use a p.V.MarshalBinary and convert it to a hex-string, and display that.

Reply:
Even better - I _did_ look in the `type Scalar`, but didn't follow to the `type Marshaling` where it's defined...